### PR TITLE
Created 2 new models for GroceryItem & GroceryItemType

### PR DESCRIPTION
**Pull request by CSWillis95**
Aligned ERD properties with C#

### CODE REVIEW

File: PantryPro.Server/Controllers/GroceryItemType.cs [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+
+namespace PantryPro.Server.Controllers
+{
+    [ApiController]
+    [Route("[controller]")]
+    public class GroceryItemTypeController : ControllerBase
+    {
+        private static readonly string[] Descriptions = new[]
+        {
+            "Int", "String", "Array", "Boolean", "Long int", "Char", "Float", "Enum"
```
Comment (dgarcia-appdev) [inline PantryPro.Server/Controllers/GroceryItemType.cs:11]: Can you change these to "fruit", "vegetable", "meat", "whole grains", "dairy" and add any other you can think of please.

**Review by dgarcia-appdev — CHANGES_REQUESTED** [review]

File: PantryPro.Server/Controllers/GroceryItemType.cs (hunk 1 above)
Comment (CSWillis95) [reply PantryPro.Server/Controllers/GroceryItemType.cs:11]: Made the changes, submitting again

**Comment by CSWillis95** [comment]
Made the changes, submitting again

**Comment by CSWillis95** [comment]
I completed it, but can't merge again.

**Pull request closed without merging**